bcgov/moh-prime
# Remove Deferred Reason on Care Setting Change

* Deferred reason blocks updates to the Site Name if the care setting was updated before submission of the site from Comm Pharm to Comm Prac

Testing Case 1:
1. Create a site as Comm Pharm with any vendor
2. Don't make any changes to Business Licence
3. Route back to Care Setting and change to Comm Prac with any vendor
4. Observe network and there should be no PUT to update business licence when no deferred reason exists

Testing Case 2:
1. Create a site as Comm Pharm with any vendor
2. Add a deferred reason on business licence upload
3. Route back to Care Setting and change to Comm Prac with any vendor
4. Observe network for PUT to remove deferred reason if it exists
5. Observe on next page GET for deferred reason of null

Test Case 3:
1. Create a site as Comm Pharm with any vendor
2. Add a deferred reason on business licence upload
3. Upload a business licence document as well
3. Route back to Care Setting and change to Comm Prac with any vendor
4. Observe network and there should be no PUT since business licences can't be updated after a document has been uploaded so the deferred reason will persist
